stable/mariadb/templates/master-statefulset.yaml

```diff
@@ -50,6 +50,18 @@ spec:
         - name: {{ . }}
       {{- end}}
       {{- end }}
+      {{- if .Values.master.persistence.enabled }}
+      initContainers:
+      - name: init-{{ template "mariadb.fullname" . }}
+        image: "{{ template "mariadb.image" . }}"
+        command: ["sh", "-c", "chown 1001 /bitnami/mariadb"]
```

Comment:
Dumb question: why `sh -c`? (I know you didn't write the code)

Reply:
I suspect he based his code on the examples at https://kubernetes.io/docs/concepts/workloads/pods/init-containers/